src/localisation/bg.json

```
@@ -2,27 +2,60 @@
     "about": "Относно",
     "aboutDescription": "Пускане на тематични песни в игровите ви страници",
     "aboutLabel": "За Game Theme Music",
+    "backupOverrides": "backupOverrides",
+    "backupOverridesLabel": "backupOverridesLabel",
+    "backupSuccessful": "backupSuccessful",
+    "backupSuccessfulDetails": "backupSuccessful",
```

```suggestion
    "backupSuccessfulDetails": "backupSuccessfulDetails",
```

(Note that this suggestion also applies to other languages.)